### PR TITLE
Fix some typos and grammar in class.md

### PR DESCRIPTION
**Pull request by Alexander-N**
Not a native speaker, but I tried to make some small improvements.

### CODE REVIEW

File: guide/src/class.md [hunk 1]
```diff
@@ -618,6 +616,10 @@ TODO: Which traits to implement (basically `PyTypeCreate: PyObjectAlloc + PyType
 
 ## How methods are implemented
 
-Users should be able to define a `#[pyclass]` with or without `#[pymethods]`, while pyo3 needs a trait with a function that returns all methods. Since it's impossible make the code generation in pyclass dependent on whether there is an impl block, we'd need to make to implement the trait on `#[pyclass]` and override the implementation in `#[pymethods]`, which is to my best knowledge only possible with the specialization feature, which is can't be used on stable.
+Users should be able to define a `#[pyclass]` with or without `#[pymethods]`, while PyO3 needs a
+trait with a function that returns all methods. Since it's impossible make the code generation in
```
Comment (kngwyu) [inline guide/src/class.md]: it's impossible make -> it's impossible **to** make ( though it's not your mistake :)

File: guide/src/class.md [hunk 2]
```diff
@@ -618,6 +616,10 @@ TODO: Which traits to implement (basically `PyTypeCreate: PyObjectAlloc + PyType
...
+pyclass dependent on whether there is an impl block, we'd need to implement the trait on
+`#[pyclass]` and override the implementation in `#[pymethods]`, which is to my best knowledge
```
Comment (kngwyu) [inline guide/src/class.md]: to my best knowledge -> to the best of my knowledge/to my knowledge

File: guide/src/class.md [hunk 3]
```diff
@@ -464,11 +463,10 @@ impl MyClass {
 
 ## Class customizations
 
-Python object model defines several protocols for different object behavior,
-like sequence, mapping or number protocols. pyo3 library defines separate trait for each
-of them. To provide specific python object behavior you need to implement specific trait
-for your struct. Important note, each protocol implementation block has to be annotated
-with `#[pyproto]` attribute.
+Python's object model defines several protocols for different object behavior, like sequence,
+mapping or number protocols. PyO3 defines a separate trait for each of them. To provide specific
```
Comment (kngwyu) [inline guide/src/class.md]: a separate trait -> separate traits

File: guide/src/class.md [hunk 4]
```diff
@@ -248,7 +249,7 @@ impl MyClass {
 In this case property `num` is defined. And it is available from python code as `self.num`.
 
 Also both `#[getter]` and `#[setter]` attributes accepts one parameter.
-If parameter is specified, it is used and property name. i.e.
+If this parameter is specified, it is used as property name. i.e.
```
Comment (kngwyu) [inline guide/src/class.md]: as property name -> as a property name

**Comment by kngwyu** [comment]
Thanks, great improvement for us.

**Comment by Alexander-N** [comment]
Thanks for the review, done.

**Comment by konstin** [comment]
Thank you!

**Pull request merged**